src/FixerDefinition/VersionSpecificCodeSampleInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerDefinition;
+
+/**
+ * @author Andreas Moeller <am@localheinz.com>
+ *
+ * @internal
```

Comment:
`CodeSampleInterface` is not internal, as it's supposed to be used by users writing custom fixers.
not sure why this, extended version, should be marked as internal ...